lang/es-419.json

```diff
@@ -217,9 +217,11 @@
         "button": "Comparar productos",
         "header": "Comparando {products, plural, one {# Product} other {# Products}}",
         "remove": "Quitar",
+        "remove_all": "Remove All",
```

Comment:
Please check all translations that were added and make them in their respective language using Google Translate